types/http.d.ts

```diff
@@ -191,13 +194,17 @@ export type HttpMethod = 'GET' | 'POST' | 'DELETE' | 'HEAD' | 'PATCH' | 'PUT' |
  */
 export type HttpRequestUserType = 'AppService' | 'StaticWebApps';
 
-export type HttpResponseBody = string | Buffer | NodeJS.ArrayBufferView | number | object;
-
-export interface HttpResponse {
+export interface HttpResponseInit {
     /**
      * HTTP response body
      */
-    body?: HttpResponseBody;
+    body?: BodyInit;
+
+    /**
+     * A JSON-serializable HTTP Response body.
+     * If set, the `HttpResponseInit.body` property will be ignored in favor of this property
+     */
+    jsonBody?: any;
```

Comment:
Rather than having an `any`, why not make it a generic, so it's not straight returned it's assigned to a variable, you can get type information

Reply:
Do you have example code of how a generic would help here? I don't like generics 😝. If they can prevent a bug at compile time that normally would only be found at runtime, then they can be worth it. But most of the time I think they just give people a false sense of security and add unnecessary fluff to the codebase

Reply:
I was thinking of if you're writing a test and you want to assert on the output, you only have an `any` so you can't easily check the fields of the `jsonBody` for asserts.